client/src/components/CharacterCreationForm.js

```diff
@@ -0,0 +1,148 @@
+import React, { useState, useEffect } from "react";
+import { Redirect } from "react-router-dom";
+import ErrorList from "./layout/ErrorList";
+import translateServerErrors from "../services/translateServerErrors";
+
+const CharacterCreationForm = ({ user }) => {
+  if (!user) {
+    return null;
+  }
+
+  const [redirect, setRedirect] = useState({
+    shouldRedirect: false,
+    charId: "",
+    hunter: "",
+  });
+
+  const [characterData, setCharacterData] = useState({
+    userId: user.id,
+    name: "",
+    hunterIndex: "",
+  });
+  const [hunterData, setHunterData] = useState([]);
+  const [errors, setErrors] = useState({});
+
+  const getHunters = async () => {
+    try {
+      const response = await fetch("/api/v1/hunters");
+      if (!response.ok) {
+        const errorMessage = `${response.status} (${response.statusText})`;
+        const error = new Error(errorMessage);
+        throw error;
+      }
+      const body = await response.json();
+      setHunterData(body.hunters);
+      setCharacterData({ ...characterData, hunterIndex: body.hunters[0].index });
+    } catch (error) {
+      console.error(`Error in fetch: ${error.message}`);
+    }
+  };
+
+  useEffect(() => {
+    getHunters();
+  }, []);
+
+  if (!hunterData) {
+    return null;
+  }
+
+  if (characterData.hunterIndex === "") {
+    return null;
+  }
```

Comment:
could we combine all of these if statements that return null, so there is one generic message?